pom.xml

```diff
@@ -618,6 +618,26 @@
                 </execution>
               </executions>
             </plugin>
+            <plugin>
+              <artifactId>maven-javadoc-plugin</artifactId>
+              <version>3.0.1</version>
+              <configuration>
+                <additionalparam>-Xdoclint:none</additionalparam>
```

Comment:
Not sure this parameter is still there, might have been replaced in `3.x`: https://maven.apache.org/plugins/maven-javadoc-plugin/jar-mojo.html#additionalOptions

Reply:
I will remove this and see what happens.